src/components/HttpRequestWidget.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Paper } from 'material-ui'
+import Divider from 'material-ui/Divider'
+import Theme from '../../style/theme'
+
+class HttpRequestWidget extends React.PureComponent {
+  constructor(props) {
+    super(props)
+    this.state = {
+      badRequests: 0,
+      numberOfRequests: 0,
+      totalResponseTime: 0,
+      maxResponseTime: 0
+    }
+  }
+
+  componentDidMount() {
+    this.processData()
```

Comment:
:+1:

Reply:
Do we want to track total response time? Don’t think this is needed. We should do average response time instead. 

Also maybe percentage of errors?

Reply:
:D total response time is just being used to later show the average response time. You're more than welcome to add any stats you think will be helpful once this merges.